packages/core/admin/admin/src/components/GlobalStyle/index.js

```diff
@@ -1,11 +1,5 @@
 import { createGlobalStyle } from 'styled-components';
 
-const loadCss = async () => {
-  await import(/* webpackChunkName: "cropper-css" */ 'cropperjs/dist/cropper.css');
```

Comment:
I wonder if moving it to the upload plugin will cause some flickering since it will be reimported each time the ML is mounted?

Reply:
👋🏼 

If we used a dynamic import, it would create a separate chunk which then could load late and lead to flickering - I agree, but we don't do that. A sync import will be bundled together with the `PreviewBox` component, and as soon as it has loaded, the styles will be there immediately. It increases the size of the chunk a bit, but since it is pretty much isolated, I don't see a big problem in this. Also: the Modal containing the `PreviewBox` could be moved into its own webpack chunk, if we would want to do this.